packages/scss/src/components/indexTable/component.scss

```diff
@@ -14,7 +14,7 @@
 	border-radius: var(--components-indexTable-border-radius);
 	background-color: var(--components-indexTable-background-color);
 	color: var(--palettes-neutral-800);
-	contain: paint;
+	contain: layout;
```

Comment:
It seems to break our curent overflow!

https://github.com/user-attachments/assets/bbd5af81-2cac-438e-81ad-5a693dfa3042